tests/cases/test_raft.rs

```diff
@@ -3978,3 +3949,47 @@ fn test_remove_learner() {
     assert!(n1.prs().nodes().is_empty());
     assert!(n1.prs().learner_nodes().is_empty());
 }
+
+#[test]
+fn test_learner_respond_vote() {
+    let mut n1 = new_test_learner_raft(1, vec![1, 2], vec![3], 10, 1, new_storage());
+    n1.become_follower(1, INVALID_ID);
+    n1.reset_randomized_election_timeout();
+
+    let mut n3 = new_test_learner_raft(3, vec![1, 2], vec![3], 10, 1, new_storage());
+    n3.become_follower(1, INVALID_ID);
+    n3.reset_randomized_election_timeout();
+
+    let timeout = n1.get_election_timeout();
+
+    let do_campaign = |nw: &mut Network| {
+        for _ in 0..timeout << 1 {
+            nw.peers.get_mut(&1).unwrap().tick();
+            nw.peers.get_mut(&3).unwrap().tick();
```

Comment:
What's the point of ticking node 3?

Reply:
To show all `Vote` messages come from store 1 even if 3 will also `step(MsgHup)`.

Reply:
I don't get it. Where are all those messages showed? How does it help with the test case?

Reply:
No, they are still here.